BD-1860: Update rate limit for User Update step (#4692)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/user_update.md

Drag and drop the component from the sidebar, or click the <i class="fas fa-plus-circle"></i> plus button at the bottom of the variant or step and select **User Update**. 

> There are three options that allow you to update existing, add new, or remove user profile information. When used in an app group, this step can update up to 200,000 user profiles per minute combined.

Reviewer: This implies the existence of user update steps outside of an app group. From the ticket it seems like its 200,000 user profiles per minute per app group. As in all the user update steps in an app group have a cumulative rate limit of 200,000.

```suggestion
There are three options that allow you to update existing, add new, or remove user profile information. All combined, the User Update steps in an app group can update up to 200,000 user profiles per minute.
```
